library/src/main/java/org/xmtp/android/library/Conversations.kt

```diff
@@ -745,14 +737,6 @@ data class Conversations(
                 }
             } catch (error: CancellationException) {
                 break
-            } catch (error: StatusException) {
-                if (error.status.code == io.grpc.Status.Code.UNAVAILABLE) {
-                    continue
-                } else {
-                    break
-                }
-            } catch (error: Exception) {
-                continue
             }
```

Comment:
@rsappelit is this kind of what you had in mind? Instead of catching these StatusExceptions just let them be thrown so you can catch and handle them?